pkg/system/sysinfo.go

```diff
@@ -3,9 +3,50 @@
 package system
 
 import (
+	"io/ioutil"
+	"os"
 	"syscall"
+
+	"golang.org/x/sys/unix"
 )
 
+// SysInfo defines system info on current machine
+type SysInfo struct {
+	AppArmor bool
+	Seccomp  bool
+
+	*CgroupInfo
+}
+
+// NewSysInfo creates a system info about current machine.
+func NewSysInfo() *SysInfo {
+	sysInfo := &SysInfo{CgroupInfo: NewCgroupInfo()}
+
+	// Check if AppArmor is supported.
+	// isAppArmorEnabled returns true if apparmor is enabled for the host.
+	// This function is forked from
+	// https://github.com/opencontainers/runc/blob/1a81e9ab1f138c091fe5c86d0883f87716088527/libcontainer/apparmor/apparmor.go
+	// to avoid the libapparmor dependency.
+	if _, err := os.Stat("/sys/kernel/security/apparmor"); err == nil && os.Getenv("container") == "" {
```

Comment:
who set `container` env?

Reply:
runc did for this, I am not sure what this stands for, we can remove this env